src/raster/r.area.createweight/r.area.createweight.py

```diff
@@ -171,7 +171,7 @@
 #%option
 #% key: param_grid
 #% type: string
-#% description: Python dictionary of customized tunegrid for sklearn RFregressor
+#% description: Python dictionary of customized tunegrid for sklearn RandomForestRegressor
```

Comment:
I went back in the scikit learn docs to correct this, but was this unique abbreviation of the concept made to adapt to GUI size constraints?

Reply:
I assume there are no problems then.

Reply:
Your change is fine.